converged_sound_open_fw_modules_development/scripts/build_library.py

```diff
@@ -0,0 +1,255 @@
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Copyright(c) 2021 Intel Corporation. All rights reserved.
+
+from shutil import copyfile
+from pathlib import Path
+import argparse
+import os
+import xml.etree.ElementTree as ET
+import sys
+import errno
+
+# usage example
+# python3 build_library.py lx7hifi4_plat aca_lib 1.0 aca_module
+
+# Global paths
+REPO_PATH = Path(__file__).absolute().parents[1]
+COMMON_BINMAP_PATH = REPO_PATH / Path("FW/intel_common/module_binmaps")
+INTEL_ADSP_DIR = REPO_PATH / Path("FW/src/intel_adsp/")

```

Comment:
This should be made vendor agnostic, or at least mark down places where vendor specific steps need to be taken like in sof/scripts/xtensa-build-all.sh .